src/itables/datatables_format.py

```diff
@@ -108,17 +111,17 @@ def datatables_rows(df, count=None, warn_on_unexpected_types=False, pure_json=Fa
             cls=generate_encoder(warn_on_unexpected_types),
             allow_nan=not pure_json,
         )
-    except AttributeError:
+    else:
         # Polars DataFrame
+        df = nw.from_native(df)
         data = list(df.iter_rows())
```

Comment:
drive-by - this would be more efficient as `data = df.rows()` (it produces the same thing, just faster)

Reply:
I did the PR for this. Thanks for the feedback.